server/src/internalClusterTest/java/org/elasticsearch/cluster/routing/allocation/decider/WriteLoadConstraintDeciderIT.java

```diff
@@ -68,6 +68,8 @@ protected Collection<Class<? extends Plugin>> getMockPlugins() {
      */
     public void testHighNodeWriteLoadPreventsNewShardAllocation() {
         int randomUtilizationThresholdPercent = randomIntBetween(50, 100);
+        int numberOfWritePoolThreads = randomIntBetween(10, 20);
+        double shardWriteLoad = randomDoubleBetween(0.0, 0.2, true);
```

Comment:
Would probably be a lot easier to reason about if shard write loads were zero (i.e. no risk of the decider preventing the movement), but also less realistic.

Reply:
Ahh of course... I changed `lowerInclusive` to `false` in ba15f8fd19d026cb4e09badbd4b9007dc064cea3 to ensure we don't get zeros